Shared/Services/LoopringGraphQLService.cs

```diff
@@ -1,23 +1,37 @@
 ﻿using Lexplorer.Helpers;
 using Lexplorer.Models;
 using Newtonsoft.Json;
-using Newtonsoft.Json.Converters;
-using Newtonsoft.Json.Serialization;
 using Newtonsoft.Json.Linq;
 using RestSharp;
 using System.Diagnostics;
-using System.Threading.Tasks;
-using System.Collections.Generic;
-using System;
-using System.Linq;
 using RestSharp.Serializers;
-using System.Globalization;
 using System.Text.RegularExpressions;
-using System.Threading;
```

Comment:
Removing these will break the unit test project as it doesn't feature something like what "_Imports.razor" does - AFAIK.

Reply:
This was removed by mistake, and has later been added if you see line 10